controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/horizon/TsdbQueryRewriterTest.java

```diff
@@ -35,6 +35,10 @@ public void rewrites_query() throws IOException {
         assertRewrite("no-filters.json",
                 "no-filters.expected.json",
                 Role.reader(TenantName.from("tenant2")), Role.athenzTenantAdmin(TenantName.from("tenant3")));
+
+        assertRewrite("meta-query.json",
+                "filters-meta-query.json",
```

Comment:
Follow the same filename pattern as the others so they are grouped together, i.e. `meta-query.expected.json`?